src/test/java/uk/sky/cqlmigrate/CqlMigratorImplTest.java

```diff
@@ -159,52 +166,52 @@ public void shouldNotRemoveLockAfterMigrationFailed() throws Exception {
         }
 
         //then
-        ResultSet resultSet = session.execute("SELECT * FROM cqlmigrate.locks WHERE name = ?", LOCK_NAME);
-        assertThat(resultSet.isExhausted()).as("Is lock released").isFalse();
+        ResultSet resultSet = session.execute(newInstance("SELECT * FROM cqlmigrate.locks WHERE name = ?", LOCK_NAME));
+        assertThat(resultSet.one()).as("Is lock released").isNotNull();
     }
 
     @Test
     public void shouldRemoveLockAfterMigrationFailedIfUnlockOnFailureIsSetToTrue() throws Exception {
         //given
         CqlMigrator migrator = new CqlMigratorImpl(CqlMigratorConfig.builder()
-            .withLockConfig(CassandraLockConfig.builder().unlockOnFailure().build())
-            .withReadConsistencyLevel(ConsistencyLevel.ALL)
-            .withWriteConsistencyLevel(ConsistencyLevel.ALL)
-            .build(), new SessionContextFactory());
+                .withLockConfig(CassandraLockConfig.builder().unlockOnFailure().build())
+                .withReadConsistencyLevel(ConsistencyLevel.ALL)
+                .withWriteConsistencyLevel(ConsistencyLevel.ALL)
+                .build(), new SessionContextFactory());
         Collection<Path> cqlPaths = singletonList(getResourcePath("cql_bootstrap_missing_semicolon"));
 
         //when
         try {
-            migrator.migrate(CASSANDRA_HOSTS, binaryPort, username, password, TEST_KEYSPACE, cqlPaths);
+            migrator.migrate(CASSANDRA_HOSTS, LOCAL_DC, binaryPort, username, password, TEST_KEYSPACE, cqlPaths);
         } catch (RuntimeException ignore) {
         }
 
         //then
-        ResultSet resultSet = session.execute("SELECT * FROM cqlmigrate.locks WHERE name = ?", LOCK_NAME);
-        assertThat(resultSet.isExhausted()).as("Is lock released").isTrue();
+        ResultSet resultSet = session.execute(newInstance("SELECT * FROM cqlmigrate.locks WHERE name = ?", LOCK_NAME));
+        assertThat(resultSet.one()).as("Is lock released").isNull();
     }
 
     @Test
     public void shouldRetryWhenAcquiringLockIfNotInitiallyAvailable() throws Exception {
         //given
-        session.execute("INSERT INTO cqlmigrate.locks (name, client) VALUES (?, ?)", LOCK_NAME, UUID.randomUUID().toString());
+        session.execute(newInstance("INSERT INTO cqlmigrate.locks (name, client) VALUES (?, ?)", LOCK_NAME, UUID.randomUUID().toString()));
         Collection<Path> cqlPaths = singletonList(getResourcePath("cql_bootstrap"));
 
         //when
         Future<?> future = executorService.submit(() -> MIGRATOR.migrate(
-                CASSANDRA_HOSTS, binaryPort, username, password, TEST_KEYSPACE, cqlPaths));
+                CASSANDRA_HOSTS, LOCAL_DC, binaryPort, username, password, TEST_KEYSPACE, cqlPaths));
         session.execute("TRUNCATE cqlmigrate.locks");
         Thread.sleep(1000);
         future.get();
 
         //then
         try {
-            cluster.connect(TEST_KEYSPACE);
+            session.getMetadata().getKeyspace(TEST_KEYSPACE);
```

Comment:
Are these checking the same thing? getKeyspace() returns an optional which could still be empty if the bootstrap hasn't been applied?